docs/tutorials/issue-tracker/08-adding-admission-control.md

```diff
@@ -52,7 +52,7 @@ type KindValidator interface {
 ```
 Basically, it consumes an admission request and produces a validation error if validation fails, or returns nil on success. 
 The `simple` package has a ready-to-go implementation for this: `simple.Validator`, which calls `ValidateFunc` when `Validate` is called. 
-That's what we're using here, but you can always define your own type to implement `KindValidator`, too. 
+We use the `simple.Validator` struct here to validate, but you can use any type that implements `simple.KindValidator`. 
```

Comment:
I couldn't get these changes to apply for me, even after a teardown & clean. But its near the end of the day, I can try again tomorrow!